<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Core::Internal::SystemSettings</class>
 <widget class="QWidget" name="Core::Internal::SystemSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>527</width>
    <height>469</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="systemBox">
     <property name="title">
      <string>System</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="3">
       <widget class="QToolButton" name="helpExternalFileBrowserButton">
        <property name="text">
         <string>?</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="resetFileBrowserButton">
        <property name="toolTip">
         <string comment="File Browser">Reset to default.</string>
        </property>
        <property name="text">
         <string>Reset</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="4">
       <widget class="QWidget" name="fileSystemCaseSensitivityWidget" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label">
           <property name="toolTip">
            <string>Influences how file names are matched to decide if they are the same.</string>
           </property>
           <property name="text">
            <string>File system case sensitivity:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="fileSystemCaseSensitivityChooser"/>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item row="5" column="0" colspan="4">
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QCheckBox" name="autoSaveCheckBox">
          <property name="text">
           <string>Auto-save modified files</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_6">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="autoSaveIntervalLabel">
          <property name="text">
           <string>Interval:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="autoSaveInterval">
          <property name="suffix">
           <string extracomment="unit for minutes">min</string>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_5">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="2" column="1" colspan="2">
       <widget class="Utils::PathChooser" name="patchChooser" native="true"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="externalFileBrowserLabel">
        <property name="text">
         <string>External file browser:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="terminalLabel">
        <property name="text">
         <string>Terminal:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="externalFileBrowserEdit"/>
      </item>
      <item row="4" column="0" colspan="4">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="modifiedLabel">
          <property name="text">
           <string>When files are externally modified:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="reloadBehavior">
          <property name="currentIndex">
           <number>0</number>
          </property>
          <item>
           <property name="text">
            <string>Always Ask</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Reload All Unchanged Editors</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Ignore Modifications</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="resetTerminalButton">
        <property name="toolTip">
         <string comment="Terminal">Reset to default.</string>
        </property>
        <property name="text">
         <string>Reset</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="terminalComboBox">
        <property name="editable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="7" column="0" colspan="4">
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QCheckBox" name="warnBeforeOpeningBigFiles">
          <property name="text">
           <string>Warn before opening text files greater than</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="bigFilesLimitSpinBox">
          <property name="suffix">
           <string>MB</string>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>500</number>
          </property>
          <property name="value">
           <number>5</number>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="patchCommandLabel">
        <property name="text">
         <string>Patch command:</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0" colspan="4">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QCheckBox" name="autoSuspendCheckBox">
          <property name="toolTip">
           <string>Automatically free resources of old documents that are not visible and not modified. They stay visible in the list of open documents.</string>
          </property>
          <property name="text">
           <string>Auto-suspend unmodified files</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_7">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="autoSuspendLabel">
          <property name="toolTip">
           <string>Minimum number of open documents that should be kept in memory. Increasing this number will lead to greater resource usage when not manually closing documents.</string>
          </property>
          <property name="text">
           <string>Files to keep open:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="autoSuspendMinDocumentCount">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>500</number>
          </property>
          <property name="value">
           <number>30</number>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>30</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Utils::PathChooser</class>
   <extends>QWidget</extends>
   <header location="global">utils/pathchooser.h</header>
   <container>1</container>
   <slots>
    <signal>editingFinished()</signal>
    <signal>browsingFinished()</signal>
   </slots>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>terminalComboBox</tabstop>
  <tabstop>resetTerminalButton</tabstop>
  <tabstop>externalFileBrowserEdit</tabstop>
  <tabstop>resetFileBrowserButton</tabstop>
  <tabstop>helpExternalFileBrowserButton</tabstop>
  <tabstop>fileSystemCaseSensitivityChooser</tabstop>
  <tabstop>reloadBehavior</tabstop>
  <tabstop>autoSaveCheckBox</tabstop>
  <tabstop>autoSaveInterval</tabstop>
  <tabstop>autoSuspendCheckBox</tabstop>
  <tabstop>autoSuspendMinDocumentCount</tabstop>
  <tabstop>warnBeforeOpeningBigFiles</tabstop>
  <tabstop>bigFilesLimitSpinBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
